adage/adage/config.py.template

```diff
@@ -7,6 +7,10 @@
 # and replace with values tailored to your deployment.
 import random
 
+import os
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
I see why this was specified now. I don't know a way around specifying where the repo will land in your deployment pipeline. With the CI providers we can depend on them to clone, and then use paths relative to that. But for the first deployment, I think a location for the destination needs to be specified. Do you want to switch back to a home_dir? If we leave the other locations relative it will aid our deployment in other contexts.